packages/now-build-utils/src/detectors/ember.ts

```diff
@@ -0,0 +1,22 @@
+import { DetectorParameters, DetectorResult } from '../types';
+
+export default async function detectEmber({
+  fs: { hasDependency },
+}: DetectorParameters): Promise<DetectorResult> {
+  const hasEmber = await hasDependency('ember-cli');
+  if (!hasEmber) return false;
+  return {
+    buildCommand: ['ember', 'build'],
+    buildDirectory: 'build',
```

Comment:
```suggestion
    buildDirectory: 'dist',
```

This is dist according to frameworks.ts